src/kit/LogViewer/LogViewer.tsx

```diff
@@ -135,6 +138,11 @@ const LogViewer: React.FC<Props> = ({
   const { refObject: logsRef, refCallback, size: containerSize } = useResize();
   const { size: pageSize } = useResize();
   const charMeasures = useGetCharMeasureInContainer(logsRef, containerSize);
+  const [logContainerSize, setLogContainerSize] = useState(() => {
+    if (height !== undefined) return height;
+
+    return pageSize.height + 250;
```

Comment:
Based on the discussion in https://github.com/determined-ai/determined/pull/8515, we don't want to specify height number.